src/cem_drive.F

```diff
@@ -552,8 +552,20 @@ subroutine time_advancing_pde
       integer ifirst,ilast
 
       if (.not.usesteps) then
-         nsteps = int(fintim/dt)
+         if (ifrestart) then
+            nsteps = int((fintim-restart_time)/dt)
+            if (nid.eq.0) then
+               write(6,*) 'fintim',fintim
```

Comment:
Can we get into the habit of also printing the routine where print statements are coming from? Otherwise it's hard to track down the source of things in the log.